imports/api/utils.js

```diff
@@ -8,6 +9,12 @@ export function toggleElementInArray(collection, id, arrayName, element) {
   collection.update(id, modifier);
 }
 
+export function toggle(element, array) {
+  if (!Array.isArray(array)) array = [];
```

Comment:
itt egy picit tul megengedoek vagyunk szerintem
Ha a 2. parameter 'undefined' akkor jogos hogy krealunk egy uj tombot,
De ha ott pl egy bool-t ad be neki a hivo (mert osszekeveri egy masik toggle nevu  fuggvennyel amit hasznalt korabban) akkor jobb lenne hibaval elszallni, mint csendben ures Array-ra konvertalni a bool-t, es ugy csinalni mintha minden rendben ment volna...